### PR TITLE
Assorted sprite ports

### PR DESCRIPTION
**Pull request by Generalcamo**
This PR replaces ancient sprites, with various ports.

The mass driver is an ancient sprite that is not 3/4ths. A new sprite was ported from /tg/station (https://github.com/tgstation/tgstation/pull/65828), which replaces the old mass driver with a smoother animation and a 3/4ths perspective. 

The table lamps were previously using a cabinet perspective. Sprite replacements were ported from CEV Eris (https://github.com/discordia-space/CEV-Eris/pull/6804) which are in a 3/4ths perspective. These sprites have in-hands, but no changes are necessary for these.

The atmos tanks are ancient, and have a very noisy sprite. Most of these were ported from CEV Eris (Commit by SpadesSlick), however there was no hydrogen tank. I created the hydrogen tank myself by using the colors from the small tank, and modifying the sprite manually.

### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| icon/atmos/tank.dmi | SpadesSlick (CEV Eris) | GNU Affero General Public License v3 |
| icons/obj/lighting.dmi | Kegdo (CEV Eris) | GNU Affero General Public License v3 |
| icons/obj/stationobjs.dmi | Wallem (/tg/station) | GNU Affero General Public License v3 |

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

File: html/changelogs/GeneralCamo - Assorted sprite ports.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - imageadd: "Replaced table lamp sprites (Credit to CEV Eris, Kegdo)"
+  - imageadd: "Replaced static air tank sprites (Credit to CEV Eris, SpadesSlick, with minor adjustments by GeneralCamo)"
+  - imageadd: "Replaced Mass Driver sprites (Credit to /tg/station, Wallem)"
```
Comment (SleepyGemmy) [inline html/changelogs/GeneralCamo - Assorted sprite ports.yml:43]:
```suggestion
  - imageadd: "Replaced table lamp sprites."
  - imageadd: "Replaced volume tank sprites."
  - imageadd: "Replaced mass driver sprites."
```
credit is already part of the PR description

Comment (Generalcamo) [reply html/changelogs/GeneralCamo - Assorted sprite ports.yml:43]: `CONTRIBUTING.md` guidelines state to put this in the PR description, and the changelog plaintext, or the `author` field.

Comment (SleepyGemmy) [reply html/changelogs/GeneralCamo - Assorted sprite ports.yml:43]: preferably put the co-authors in the author field, then.

Comment (SleepyGemmy) [inline html/changelogs/GeneralCamo - Assorted sprite ports.yml:43]:
```suggestion
  - imageadd: "Replaced table lamp sprites. (Credit to CEV Eris, Kegdo.)"
  - imageadd: "Replaced volume tank sprites. (Credit to CEV Eris, SpadesSlick, with minor adjustments by GeneralCamo.)"
  - imageadd: "Replaced mass driver sprites. (Credit to /tg/station, Wallem.)"
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**